include/cloysterhpc/answerfile.h

```diff
@@ -123,6 +127,7 @@ class AnswerFile {
 
     void loadFile(const std::filesystem::path& path);
     std::vector<std::shared_ptr<ITool>> getTools();
+    std::vector<std::shared_ptr<IService>> getServices();
```

Comment:
Same question regarding the `std::shared_ptr`.

Reply:
I had the feeling we would use it later in different ways, at least when I imagined it and created. For now it's a very simple implementation only to install and configure the tools in a easier way. That's why I put shared.